meshroom/ui/qml/GraphEditor/GraphEditor.qml

```diff
@@ -799,8 +799,8 @@ Item {
     function fit() {
         // compute bounding box
         var bbox = boundingBox()
-        // rescale
-        draggable.scale = Math.min(root.width/bbox.width, root.height/bbox.height)
+        // rescale, if low number of node, zomm 100% to prevent huge text
+        draggable.scale = nodeRepeater.count < 4 ? 1 :  Math.min(root.width/bbox.width, root.height/bbox.height)
```

Comment:
Not sure a threshold on the number of nodes is the right approach here as it doesn't take the graph layout into account and can lead to unexpected situations when we are right below/above the threshold.

Maybe try imposing a limit on the min/max scale instead ?

Reply:
zoom

Reply:
> Not sure a threshold on the number of nodes is the right approach here as it doesn't take the graph layout into account and can lead to unexpected situations when we are right below/above the threshold.
> 
> Maybe try imposing a limit on the min/max scale instead ?

Done, the max is now 120% !

> zoom

Done :face_with_spiral_eyes: